? .deps
? .libs
? config.guess
? config.h
? config.sub
? glib-1.2.8
? libtool
? ltmain.sh
? stamp-h
? stamp-h.in
Index: ChangeLog
===================================================================
RCS file: /home/freedesktop/pkgconfig/ChangeLog,v
retrieving revision 1.42
diff -u -r1.42 ChangeLog
--- ChangeLog	19 Sep 2002 23:59:12 -0000	1.42
+++ ChangeLog	25 Sep 2002 22:04:10 -0000
@@ -1,3 +1,8 @@
+2002-09-26  Anders Carlsson  <andersca@gnu.org>
+
+	* pkg.c (verify_package): Use strncmp when checking for
+	the -I prefix.
+
 2002-09-19  Havoc Pennington  <hp@pobox.com>
 
 	* configure.in: 0.13
Index: pkg.c
===================================================================
RCS file: /home/freedesktop/pkgconfig/pkg.c,v
retrieving revision 1.22
diff -u -r1.22 pkg.c
--- pkg.c	19 Sep 2002 14:23:55 -0000	1.22
+++ pkg.c	25 Sep 2002 22:04:10 -0000
@@ -801,8 +801,8 @@
       /* we put things in canonical -I/usr/include (vs. -I /usr/include) format,
        * but if someone changes it later we may as well be robust
        */
-      if (((strcmp (iter->data, "-I") == 0) && (offset = 2))||
-          ((strcmp (iter->data, "-I ") == 0) && (offset = 3)))
+      if (((strncmp (iter->data, "-I", 2) == 0) && (offset = 2))||
+          ((strncmp (iter->data, "-I ", 3) == 0) && (offset = 3)))
         {
 	  if (offset == 0)
 	    {
